include/menus/main_menu.sh

```
@@ -11,31 +11,41 @@ main_menu() {
+    \e[49m     \e[38;5;81;48;5;74m▄\e[38;5;80;48;5;117m▄\e[49m \e[38;5;116;49m▄\e[38;5;74;48;5;81m▄▄\e[49m       \e[38;5;74;48;5;81m▄\e[38;5;117;48;5;117m▄\e[49m  \e[38;5;81;48;5;75m▄\e[38;5;117;48;5;74m▄\e[49m        \e[48;5;175m \e[38;5;168;48;5;168m▄\e[49m \e[38;5;168;48;5;168m▄\e[48;5;175m \e[49m                    \e[38;5;221;48;5;11m▄\e[38;5;227;48;5;220m▄\e[49m   \e[49;38;5;220m▀▀▀▀▀▀▀\e[49m   \e[38;5;220;49m▄\e[38;5;227;48;5;11m▄\e[49;38;5;220m▀\e[49m     \e[m
+    \e[49m     \e[38;5;116;48;5;74m▄\e[38;5;81;48;5;74m▄\e[38;5;81;48;5;81m▄\e[38;5;81;48;5;75m▄\e[38;5;74;48;5;117m▄\e[49m         \e[38;5;153;48;5;117m▄\e[38;5;81;48;5;81m▄▄\e[38;5;81;48;5;74m▄\e[38;5;74;48;5;117m▄\e[49m        \e[38;5;169;48;5;175m▄\e[38;5;169;48;5;169m▄▄▄\e[38;5;168;48;5;175m▄\e[49m                    \e[49;38;5;220m▀\e[49;38;5;227m▀\e[38;5;220;48;5;227m▄\e[38;5;11;48;5;11m▄▄▄▄▄▄▄▄▄▄▄\e[38;5;220;48;5;11m▄\e[49;38;5;227m▀\e[49;38;5;220m▀\e[49m      \e[m
+    \e[49m                                                                                \e[m
+    \e[49m \e[38;5;61;48;5;61m▄\e[38;5;110;48;5;61m▄\e[38;5;146;48;5;61m▄▄\e[38;5;146;48;5;67m▄▄\e[38;5;152;48;5;67m▄▄\e[38;5;152;48;5;73m▄▄▄▄\e[38;5;152;48;5;109m▄\e[38;5;152;48;5;115m▄▄▄\e[38;5;152;48;5;114m▄\e[38;5;152;48;5;108m▄▄\e[38;5;151;48;5;108m▄▄▄\e[38;5;151;48;5;107m▄\e[38;5;151;48;5;71m▄▄▄\e[38;5;151;48;5;107m▄▄\e[38;5;187;48;5;113m▄▄\e[38;5;186;48;5;149m▄▄\e[38;5;186;48;5;148m▄\e[38;5;192;48;5;148m▄\e[38;5;192;48;5;184m▄▄\e[38;5;228;48;5;184m▄▄\e[38;5;228;48;5;11m▄▄\e[38;5;228;48;5;220m▄▄▄\e[38;5;222;48;5;220m▄▄\e[38;5;222;48;5;214m▄▄▄▄▄▄\e[38;5;222;48;5;208m▄▄▄\e[38;5;216;48;5;209m▄\e[38;5;217;48;5;209m▄\e[38;5;217;48;5;203m▄▄▄▄▄\e[38;5;217;48;5;204m▄▄\e[38;5;211;48;5;204m▄\e[38;5;212;48;5;162m▄▄▄\e[38;5;212;48;5;168m▄\e[38;5;176;48;5;168m▄▄\e[38;5;182;48;5;132m▄▄▄▄▄▄\e[38;5;182;48;5;97m▄\e[38;5;97;49m▄\e[49m \e[m
+    \e[49m \e[49;38;5;61m▀▀▀▀\e[49;38;5;67m▀▀▀▀\e[49;38;5;73m▀▀▀▀\e[49;38;5;79m▀\e[49;38;5;115m▀▀▀▀\e[49;38;5;114m▀\e[49;38;5;78m▀▀▀\e[49;38;5;71m▀▀▀▀▀\e[49;38;5;107m▀\e[49;38;5;113m▀▀▀\e[49;38;5;149m▀▀\e[49;38;5;148m▀▀\e[49;38;5;184m▀▀▀▀\e[49;38;5;11m▀▀\e[49;38;5;220m▀▀▀▀▀▀\e[49;38;5;214m▀▀▀▀▀\e[49;38;5;208m▀▀▀\e[49;38;5;209m▀▀\e[49;38;5;203m▀▀▀▀▀▀\e[49;38;5;204m▀▀\e[49;38;5;198m▀\e[49;38;5;162m▀▀▀\e[49;38;5;168m▀▀▀\e[49;38;5;132m▀▀▀\e[49;38;5;133m▀▀\e[49;38;5;97m▀▀\e[49m \e[m
+    \e[49m
+    Github: https://github.com/AFCProject || Documentation: https://www.afcproject.dev/\n
+    ";
+    printf "This installation script will install/update the AFC Klipper extension to your system.\n\n"
     if [ "$files_updated_or_installed" == "True" ]; then
-      printf "\e[48;5;143m \e[49m \e[48;5;29m    \e[49m        \e[48;5;29m     \e[49m        \e[48;5;29m    \e[49m \e[48;5;143m \e[m  Prior AFC-Klipper-Add-On installation detected: $GREEN%s$RESET\n" $files_updated_or_installed
+      printf "    Prior AFC-Klipper-Add-On installation detected: $GREEN%s$RESET\n" $files_updated_or_installed
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)